go.mod

```diff
@@ -28,6 +28,11 @@ require (
 	sigs.k8s.io/controller-runtime v0.11.1
 )
 
+require (
```

Comment:
this is a strange format for go.mod - three require blocks?

Reply:
I don't know why the Go toolchain does this currently, perhaps its a bug upstream. Either way, I've given up trying to make it consolidate.

I've corrected it here but know its not a manually introduced error.